source/administration/config-settings.rst

```diff
@@ -1,6 +1,9 @@
 Configuration Settings
 ======================
 
+.. note::
+   The order of the configuration settings below are reflective of a reorganization of the System Console in v5.12 released on June 16th, 2019. To view the configuration settings based on the organization of the System Console in versions prior to v5.12, please see this `documentation <https://docs.mattermost.com/administration/prev-config-settings.rst>`_.
```

Comment:
Some of the links on this page might not be formatted correctly.

Reply:
This is now rending correctly